libnmstate/nm/ovs.py

```diff
@@ -123,6 +123,10 @@ def is_ovs_interface_type_id(type_id):
     return type_id == nmclient.NM.DeviceType.OVS_INTERFACE
 
 
+def get_bridge_info(bridge_device, devices_info):
+    return {'bridge': get_ovs_info(bridge_device, devices_info)}
```

Comment:
This seems like it should use `CONFIG_SUBTREE` instead of `'bridge'`.

Reply:
Done